doc/tutorials/get-started.md

```diff
@@ -0,0 +1,119 @@
+updates, and the map view scale. Also included in the **Status bar** is a quick
+search bar and a button to open a **Log Messages** window to view log messages.
+
+
+```{note}
+The [QGIS User
+Manual](https://docs.qgis.org/3.16/en/docs/user_manual/index.html) provides a
+detailed and comprehensive overview of QGIS' core features. Users who are new to
+Geographic Information Systems may also benefit from reviewing QGIS's [Gentle
+Introduction to
+GIS](https://docs.qgis.org/3.16/en/docs/gentle_gis_introduction/index.html)
+```
+
+## 4) Browse data in QGreenland
+
+Layers in QGreenland are organized into layer groups by category. For example,
+the "Background boundary" layer is "QGreenland boundaries" group, which is
+itself inside of the "Reference" group ("Background boundary/Reference").
+
+![layer_groups](/_images/layer_groups.jpg)
+
+### Toggle layer visibility
+
+Some layers are turned on by default when opening QGreenland. In order to
+visualize another data layer (or remove an existing one), toggle the checkbox
+next to the layer in the **Layers panel**.
+
+Toggle on the "Ice thickness (150m)" layer, which is in the "Terrain
+models/Bedmachine" group. The **Map view** should now include a visualization of
+Ice thickness at a 150m spatial resolution.
+
+![ice_thickness_displayed](/_images/ice_thickness_displayed.jpg)
+
+
+```{note}
+The search bar, located in the **Status bar**, can also be used to find layers
+in the **Layers panel**.
+![search_bar](/_images/search_bar.jpg)
+```
+
+### Navigate the Map View
+
+The **Map view** can be interacted with using a combination of the mouse and the
+**Map Navigation Toolbar**
+
+![map_navigation_toolbar](/_images/map_navigation_toolbar.jpg)
+
+By default, the **Pan Map** tool is selected
+![pan_map](/_images/pan_map.jpg). This tool allows the user to click and drag on
+the map view to change the extent. Try using this tool to pan around the map and
+explore the geography surrounding Greenland.
+
+The zoom tools ![zoom_in_out_icons](/_images/zoom_in_out_icons.jpg) can be used
+to zoom in and out of the map view. Select the **Zoom In** tool and use it to
+draw a box around southern Greenland to get a better look at the "Ice thickness
+(150m)" layer in that part of Greenland. Now select the **Pan Map** tool again
+and explore the layer in detail.
+
+```{note}
+The scroll wheel on your mouse or track pad can also be used to zoom in and out in the **Map view**
+```
+
+Hovering over the other map navigation tool icons will provide a tool tip
+indicating what the tool is used for. Try hovering over some of the other icons
+in the **Map Navigation Toolbar** to read their tooltips. Try out some of these
+other tools to explore the map and QGreenland's various data layers. See the
+[QGIS documentation on zooming and
+panning](https://docs.qgis.org/3.16/en/docs/user_manual/introduction/general_tools.html#zooming-and-panning)
+for more detailed information on how to effectively navigate the **Map View**.
```

Comment:
Let's add a Summary sub-heading and review what the user learned at the end. The final bullet in this section of the Diataxis docs specifically advises against "You have learned" language, though.

https://diataxis.fr/tutorials/#the-language-of-tutorials

Reply:
f6927b87